src/SourceLocator/SourceStubber/PhpStormStubsSourceStubber.php

```diff
@@ -107,6 +115,29 @@ public function generateFunctionStub(string $functionName) : ?StubData
         return new StubData($this->createStub($this->functionNodes[$functionName]), $this->getExtensionFromFilePath($filePath));
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    public function generateConstantStub(string $constantName) : ?StubData
+    {
+        // Needs fixes in JetBrains/phpstorm-stubs
+        if (in_array($constantName, ['TRUE', 'FALSE', 'NULL'])) {
+            $constantName = strtolower($constantName);
+        }
+
+        if (! array_key_exists($constantName, PhpStormStubsMap::CONSTANTS)) {
+            return null;
+        }
+
+        $filePath = PhpStormStubsMap::CONSTANTS[$constantName];
+
+        if (! array_key_exists($constantName, $this->constantNodes)) {
+            $this->parseFile($filePath);
+        }
+
+        return new StubData($this->createStub($this->constantNodes[$constantName]), $this->getExtensionFromFilePath($filePath));
```

Comment:
Hm this is failing with `STDIN` and I suppose `STDOUT` and `STDERR` as well. This might be due to them being resources (e.g. `fopen('php://stderr', 'w')`)  as opposed to constant values

Reply:
Yes, see line 132. It´s not possible to support them

Reply:
Just saw that. Is there a way to bail out with at least a more explicit message than a cryptic `TypeError` in this case?

Reply:
I should ignore these constants in the generate-map script. It should solve everything in much simpler way.

Reply:
Should be fixed.